### ISSUE TITLE
Feature: Don't suspend specific tab even after refresh/page change

### ISSUE THREAD

**Issue by benedictleejh** [deanoemcke/thegreatsuspender#634]
In a way, related to #493 and #473, but instead of a temporary disabling of suspension until a page refresh or URL change, the disabling would remain even when the page refreshes, or the URL changes. Because sometimes, all you want is a specific tab to not be suspended (working on something, etc.), but whitelisting domains is far too coarse for this, because you may navigate away from the original page.

I guess it would be fine if the disabling of suspension lasted until the browser closed, though if it persisted through that, it would be great.

**Comment by fwextensions** [comment]
The *Don't suspend for now* option may do what you want, though not through a restart, I think.

**Comment by benedictleejh** [comment]
The "Don't suspend for now" option does _not_ keep suspension disabling through a page refresh or URL change, hence the issue.

**Comment by deanoemcke** [comment]
I agree that this might be a nice improvement to the 'Dont suspend for now' option.
It could perhaps be renamed to 'Dont suspend this tab'.

The original use-case for this option was when listening to music through youtube and you didn't want the tab to suspend in the background. (This has since been superceeded by the 'dont suspend tabs playing audio' option).

I'm not sure how many people use this option now, and what they use it for. It would be good to know this, and if it turns out most people want the tab to be more permanently exempt, I will advocate for changing it.

**Comment by liamjohnston** [comment]
I use the heck out of 'Don't suspend for now'.

Common use case: On a video site like Udemy, where I am in the middle of a long video, which I **pause** to come back to later and not lose my place. But I also want that site to auto-suspend other times (e.g. when I'm between videos).

I suspect other users of this feature would use it for the same reason - anyone? 

I'm perfectly happy with how it works currently, though having it keep working through a browser restart would be even better. I hardly ever restart though so I don't really care personally.

**Comment by kankaristo** [comment]
I'd also prefer for this to persist through a refresh.

Somewhat related: the "Don't suspend for now" button becomes unavailable if the tab is playing audio, which is a bit annoying, because I have to pause the audio/video, and wait a few seconds before I can pause suspension.

Also somewhat related: hotkey (https://github.com/deanoemcke/thegreatsuspender/issues/208) would be nice, and would fix the above issue too.

**Comment by deanoemcke** [comment]
@kankaristo I have addressed your issue above (for the audio issue), and also implemented the hotkey in the upcoming release.

**Comment by kankaristo** [comment]
@deanoemcke, fantastic, thanks! :+1:
